### PR TITLE
[APB-2010][AK] added monitoring to 3 connectors, edited 1 connector's monitoring to fit more readable pattern by service names added appropriate tests

### PR DESCRIPTION
**Pull request by arkowalski**

### CODE REVIEW

**Comment by arkowalski** [comment]
Squash just to the last commit message

File: app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala [hunk 1]
```diff
@@ -34,7 +34,7 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
   def hasAcceptableNumberOfClients(regime: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
-    monitor(s"ConsumedAPI-AgentAssurance-hasAcceptableNumberOfClients-GET") {
+    monitor(s"ConsumedAPI-Agent-Assurance-hasAcceptableNumberOfClients-GET") {
```
Comment (arturopala) [inline app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala]: we should not change names of existing metrics, it breaks history and current dashboards

File: app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala [hunk 2]
```diff
@@ -45,7 +45,7 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
   }
 
   def getActiveCesaRelationship(url: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
-    monitor(s"ConsumedAPI-AgentAssurance-getActiveCesaRelationship-GET") {
+    monitor(s"ConsumedAPI-Agent-Assurance-getActiveCesaRelationship-GET") {
```
Comment (arturopala) [inline app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala]: as above

File: app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala (hunk 1 above)
Comment (arkowalski) [reply app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala]: OK, Ill change it back

**Pull request merged**